### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the password expiration policy section.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8585

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -49,18 +49,16 @@ If you're a user, you don't have the permissions to set your password to never e
 
 Follow the steps below if you want to set user passwords to expire after a specific amount of time.
 
-1. In the Microsoft 365 admin center, go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a> under **Org Settings**.
+1. In the Microsoft 365 admin center, go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a> .
 
     If you aren't a global admin or security admin, you won't see the Security and privacy option.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md:54]:
```suggestion
    If you aren't a global admin or security admin, you won't see the Security & privacy option.
```

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 2]
```diff
@@ -49,18 +49,16 @@ If you're a user, you don't have the permissions to set your password to never e
 
 Follow the steps below if you want to set user passwords to expire after a specific amount of time.
 
-1. In the Microsoft 365 admin center, go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a> under **Org Settings**.
+1. In the Microsoft 365 admin center, go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a> .
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
1. In the Microsoft 365 admin center, go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a>.
```

File: microsoft-365/admin/manage/set-password-expiration-policy.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
    If you aren't a global admin or security admin, you won't see the Security & privacy option.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**